exercises/E-conditionals/exercise.js

```diff
@@ -6,9 +6,14 @@
   If Daniel is a student, print out "Hi, I'm Daniel, I'm a student."
 */
 
-var name = "Daniel";
-var danielsRole = "mentor";
+const name = "Daniel";
+const danielsRole = "mentor";
 
+if (danielsRole === "mentor") {
+  console.log("Hi, I'm Daniel, I'm a mentor.");
+} else {
+  console.log("Hi, I'm Daniel, I'm a student.");
+}
```

Comment:
Nice to see that both of us came up with a different way of doing this simple exercise. Here is how I did it just to share different solutions:

const name = "Daniel";
const danielsRole = "mentor";

function classRole(danielsRole) {
  if (danielsRole === "mentor") {
    return "I'm a mentor.";
  } else return "I'm a student.";
}

console.log("Hi, I'm " + name + ", " + classRole(danielsRole));

Reply:
I agree with Ash-97. I did this like Ash and it is great seeing you did it in different way.